src/server/index.js

```diff
@@ -18,7 +18,7 @@ class Server {
     // Create Hapi server instance
     const server = Hapi.server({
       host: process.env.HOST || '0.0.0.0',
-      port: process.env.PORT || 3000,
+      port: 3000,
```

Comment:
Why are you removing `process.env.PORT`?

Reply:
updated the description just now

Reply:
if we have to use a port number we should use a different name than PORT. As nginx will use the heroku PORT